src/main/resources/views/CommercialVehicles/IVA30.hbs

```diff
@@ -1,131 +1,236 @@
 
-                </ul>
-            </div>
+    <!-- put details table here -->
+    <table class="vehicle-detail-table">
+        <thead>
+            <th scope="row" colspan="2" class="section-title-border">
+                <h1> VEHICLE DETAILS </h1>
+            </th>
+        </thead>
+        <tr class="table-row-cell">
+            <td class="table-row-cell"><b>Vehicle/Trailer NR No: </b> <!--Put data here --> </td>
+            <td class="table-row-cell"><b>Test Category: </b> <!--Put data here --> </td>
+        </tr>
+        <tr class="table-row-cell">
+            <td class="table-row-cell"><b>Make/Model: </b> <!--Put data here --> </td>
+            <td class="table-row-cell"><b>VIN/Chassis No: </b> <!--Put data here --> </td>
+        </tr>
+        <tr class="table-row-cell">
+            <td class="table-row-cell"> <b>Body Type: </b> <!--Put data here --> </td>
+            <td></td>
+        </tr>
+    </table>
+
+    <!-- Required Standards Not Met Section -->
+    <div class="section-border">
+        <div class="section-title-border">
+            <h1> DEFECT DETAILS </h1>
         </div>
-        <!-- second box -->
-        <div class="section-border">
-            <div class="section-title-border">
-                <h1>USE OF THE VEHICLE</h1>
-            </div>
-            <div class="section-text-wrapper">
-                <p id="no-margin-p">It is an offence to use a vehicle that requires type approval without an Individual Approval
-                    Certificate confirming compliance with the type approval requirements, except for the purposes of:</p>
-                <ul class="vehicle-section-list">
-                    <li> - submitting it, by previous arrangement, for an approval examination;</li>
-                    <li> - bringing it away from such an examination;</li>
-                    <li> - delivering it, by previous arrangement, to a place where relevant work is to be done on it following
-                        receipt of a refusal to issue an Individual Approval Certificate;</li>
-                    <li> - bringing it away from a place where relevant work has been done on it.</li>
+        <div class="section-title-border" id="defect-details-subtitle-border">
+            <h1> Required Standard(s) Not Met: </h1>
+        </div>
+        <div class="section-text-wrapper">
+            <!-- List incoming defects here -->
+        </div>
+    </div>
+
+    <p> I hereby refuse an Individual Approval Certificate in respect of the above described vehicle on the
+        grounds that the vehicle fails to comply with the relevant requirements prescribed under The Road Vehicles
+        (Approval) Regulations 2020 </p>
+
+    <!-- Date/name table here -->
+    <table class="inspector-detail-table">
+        <tr>
+            <td></td>
+            <td class="table-row-cell"> <b>Reapplication Required By: </b> <!--Put data here --> </td>
```

Comment:
Alignment for these boxes is off. Original cert has more width for Signed: and Name:, less width for Reapplication Required By:, Date: and Station: